tests/nopanic.ci

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env python3
 
+'''
+To prevent CI failing for approved instance of banned words, add a comment: #[allow_ci]
```

Comment:
Also, the more I think about it, it may be more readable to call this something like `#[allow_panic]`.

Reply:
I see, I was thinking of tagging it on the same line , we do the same in bandit:

https://github.com/PyCQA/bandit/blob/9b4cf91e51b2cea6754bbb80f85bf968c92d4201/doc/source/config.rst#suppressing-individual-lines

I am not of any strong view though, it could go underneath - but then there could be an instance of two (or more) sequential lines  having unwraps.